src/modules/core/producer_colour.c

```diff
@@ -251,7 +251,26 @@ static int producer_get_frame( mlt_producer producer, mlt_frame_ptr frame, int i
 		// colour is an alias for resource
 		if ( mlt_properties_get( producer_props, "colour" ) != NULL )
 			mlt_properties_set( producer_props, "resource", mlt_properties_get( producer_props, "colour" ) );
-		
+
+		char *colorstring = mlt_properties_get( producer_props, "resource" );
+		if ( colorstring && strchr( colorstring, '/' ) )
+		{
+			colorstring = strdup( strrchr( colorstring, '/' ) + 1 );
+			mlt_properties_set( producer_props, "resource", colorstring );
+			free( colorstring );
+		}
```

Comment:
Since we have this code here now, we no longer need it above in the get_image function.